### DISCUSSION TITLE
Homozygous reference variants

### DISCUSSION THREAD

**Discussion by kkmarkel** [KarchinLab/open-cravat#239] [Q&A] [answered]
Hello!

I have a question - does Open-CRAVAT annotate homozygous reference (0/0) variants?
Is there any way to include them into annotation results?

**Comment by jasminebro** [comment, marked as answer]
Hi @kkmarkel. OpenCravat does not annotate 0/0 variants. If you are using a local installation of OpenCravat, you can look at the input_filename_vcf.err file in your job directory to see which lines were skipped. 

A work around could be to edit the vcf and add a fake sample that 1/1 for every variant.
